src/js/tabs/motors.js

```diff
@@ -703,34 +936,89 @@ TABS.motors.initialize = function (callback) {
                         telemetryText += i18n.getMessage('motorsESCTemperature', {motorsESCTempValue: escTemperature});
                     }
 
-                    $('.motor_testing .telemetry .motor-' + i).html(telemetryText);
+                    $(`.motor_testing .telemetry .motor-${i}`).html(telemetryText);
                 }
             }
 
-            // servo indicators are still using old (not flexible block scale), it will be changed in the future accordingly
-            for (let i = 0; i < FC.SERVO_DATA.length; i++) {
-                const data = FC.SERVO_DATA[i] - 1000,
-                marginTop = blockHeight - (data * (blockHeight / 1000)).clamp(0, blockHeight),
-                height = (data * (blockHeight / 1000)).clamp(0, blockHeight),
-                color = parseInt(data * 0.009);
-
-                $(`.servo-${i} .label`, servos_wrapper).text(FC.SERVO_DATA[i]);
-                $(`.servo-${i} .indicator`, servos_wrapper).css({
-                    'margin-top' : `${marginTop}px`,
-                    'height' : `${height}px`,
-                    'background-color' : `rgba(255,187,0,1${color})`,
-                });
-            }
             //keep the following here so at least we get a visual cue of our motor setup
             update_arm_status();
 
             if (previousArmState != self.armed) {
                 console.log('arm state change detected');
 
-                $('#motorsEnableTestMode').change();
+                motorsEnableTestModeElement.change();
             }
         }
 
+        $('a.save').on('click', function() {
```

Comment:
Since we are on the motors tab, what about making sure the motors are stopped before saving the motor configuration and rebooting the flight controller?

Reply:
Added disarming before saving

Reply:
Sorry, I think I am overlooking something - where do you stop the motors?

Reply:
`setArmingEnabled()` will only disable arming and disarm (if armed). This does not affect motors spinning based on slider settings at all - see the handler for `#motorsEnableTestMode` for what needs to be done.